pkg/cloudprovider/fake/controller-kwok/main.go

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
I'm assuming that there is some follow-up work that is getting proposed here that would eventually see us pushing the chart and an image of the cloudprovider to some registry that everyone can access without having to checkout this repo?

Reply:
I see value in this as a developer tool, and I see additional value in users who just want to try out Karpenter. I'm not sure if we this second use-case is as common, as new Karpenter users should probably test on a real cloud provider. Thoughts on why you think we should be pushing this up to some registry?

Reply:
> Thoughts on why you think we should be pushing this up to some registry

There's potentially some cool use-cases that we could support here to run kwok with karpenter when building other controllers where you need to mock a real environment with pod and node scaling. This is definitely a musing more than anything else.